src/pages/registration.tsx

```diff
@@ -1,10 +1,17 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
-import Button from '../components/button'
-import { colors, shadows } from '../shared/constants';
-import darkDrop from '../assets/img/drop-dark.svg';
-
-const theme = 'dark';
+import React, { useEffect, useReducer, useState } from "react";
+import styled, { css } from "styled-components";
+import Button from "../components/Button";
+import { colors, Desktop, Mobile, shadows } from "../shared/constants";
+import darkDrop from "../assets/img/drop-dark.svg";
+import lightDrop from "../assets/img/drop-light.svg";
+import Input from "../components/Input";
+import Dropdown from "../components/Dropdown";
```

Comment:
```suggestion
import Button from "../components/button";
import { colors, Desktop, Mobile, shadows } from "../shared/constants";
import darkDrop from "../assets/img/drop-dark.svg";
import lightDrop from "../assets/img/drop-light.svg";
import Input from "../components/Input";
import Dropdown from "../components/dropdown";
```
This doesn't compile at the moment.

Reply:
Updated